components/AssetList/index.tsx

```diff
@@ -75,7 +96,26 @@ const AssetList = ({ list = [] }: { list: UdtList }) => {
                   {t(item.udt.type === GraphQLSchema.UdtType.Native ? 'native' : 'bridged')}
                 </td>
                 <td>
-                  <Amount amount={item.value} udt={item.udt} />
+                  {isShowUsd ? (
+                    <Tooltip
```

Comment:
Tooltip could be hidden if `item.udt.token_exchange_rate` is empty because the price is definitely not updated